system_test.sh

```diff
@@ -0,0 +1,38 @@
+#!/bin/bash
+set -exo pipefail
+
+source ./helpers.sh
+
+remove_containers
+
+case "$1" in
+    "sqlite3" )
+    rm -fr /tmp/fn_integration_tests.db
```

Comment:
I'd pick a different directory for this (this is same as in test.sh, which may collide in parallel runs.) (We probably should convert these to use mktemp in the long run.)

Reply:
I meant api_test.sh